brainstormforce/ultimate-addons-for-gutenberg
# Fix: Info-Box - Acceptance Testing 2.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/4a70e27c-773e-4536-992c-78f25fc2fada?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- Screencast for each fixes is added in card.

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Backword user 
  - Drag and drop info-box in master made some changes in padding/margin for Icon, Image, Button, Bottom spacing.
  - Switch to add info-box-atr2 branch & check the front as well as the editor. ( It does not break anything ).
- New User
  - Drag and drop block.
  - Check issue and improvement suggested by the tester is fixed or not.

### Checklist:
